code/modules/clothing/under/shorts.dm

```diff
@@ -270,4 +270,10 @@
 	desc = "A pair of ripped black jeans. The brown belt is an interesting touch."
 	icon_state = "jeansblackripped"
 	item_state = "jeansblackripped"
-	body_parts_covered = LOWER_TORSO
\ No newline at end of file
+	body_parts_covered = LOWER_TORSO
+
+/obj/item/clothing/under/pants/flared
+	name = "flared pants"
+	desc = "The peak of Bieselite fashion, these pants are flared at the ankle."
```

Comment:
```suggestion
	desc = "The peak of Biesellite fashion, these pants are flared at the ankle."
```

Reply:
deadass this is how its spelled? that sucks